tests/test_mlflow_converters.py

```diff
@@ -0,0 +1,110 @@
+from copy import copy
+from datetime import datetime
+from uuid import uuid4
+
+from pytz import UTC
+
+import pytest
+
+from mlflow.entities import Run, RunData, RunInfo, RunStatus, LifecycleStage
+
+from faculty.clients.experiment import (
+    ExperimentRun as FacultyExperimentRun,
+    ExperimentRunStatus as FacultyExperimentRunStatus,
+)
+
+from mlflow_faculty.mlflow_converters import faculty_run_to_mlflow_run
+from mlflow_faculty.py23 import to_timestamp
+
+EXPERIMENT_RUN_ID = uuid4()
+RUN_STARTED_AT = datetime(2018, 3, 10, 11, 39, 12, 110000, tzinfo=UTC)
+RUN_STARTED_AT_INT = to_timestamp(RUN_STARTED_AT) * 1000
+
+FACULTY_RUN = FacultyExperimentRun(
+    id=EXPERIMENT_RUN_ID,
+    experiment_id=661,
+    artifact_location="faculty:",
+    status=FacultyExperimentRunStatus.RUNNING,
+    started_at=RUN_STARTED_AT,
+    ended_at=None,
+    deleted_at=None,
+)
+
+
+EXPECTED_RUN_INFO = RunInfo(
+    EXPERIMENT_RUN_ID,
+    661,
+    "",  # name
```

Comment:
I know it's probably not relevant to this PR, but why does the faculty -> mlflow converter have a lot of these empty strings? Do we not have access to values for these fields at all?  Is the plan to have e.g. `user_id` available in future?

Reply:
Good question. These values are stored in tags, which are currently not returned by the backend.

In milestone 1.3, we'll add tags to resolve these.

E.g. `entry_point = run.tags.get("mlflow.project.entryPoint")`

The tags we'll add will include:

```
"mlflow.runName" -> string
"mlflow.parentRunId" -> UUID | null
"mlflow.source.type" -> "local | notebook | job | project | unknown"
"mlflow.source.name" -> "path-to-notebook"
"mlflow.source.version" -> "git-hash" | null
"mlflow.gitRepoURL" -> string | null
"mlflow.gitBranchName" -> string | null
"mlflow.project.entryPoint" -> "name-of-entry-point" | null
"mlflow.faculty.created.by" -> "job" | "user"
"mlflow.faculty.created.user.userId" -> UUID | null
"mlflow.faculty.created.job.jobId" -> UUID | null
"mlflow.faculty.created.job.runId" -> UUID | null
"mlflow.faculty.created.job.subrunId" -> UUID | null
"mlflow.user" -> string | null
```

Reply:
I see, thanks!